Gemfile

```diff
@@ -0,0 +1,40 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.1.4'
+gem 'bcrypt', '~> 3.1.7'
+gem 'bootsnap', require: false
+gem 'cancancan'
+gem 'devise'
+gem 'ffi'
+gem 'importmap-rails'
+gem 'jbuilder'
+gem 'pg', '~> 1.1'
+gem 'puma', '~> 5.0'
+gem 'rails', '~> 7.0.3'
+gem 'rubocop', require: false
+gem 'sprockets-rails'
+gem 'stimulus-rails'
+gem 'turbo-rails'
+gem 'tzinfo-data'
+
+# Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
+# gem "image_processing", "~> 1.2"
+
+group :development, :test do
+  # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
+  gem 'database_cleaner'
+  gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'rails-controller-testing'
+  gem 'rspec-rails', '~> 6.0.0.rc1'
```

Comment:
- [ ] You did super well adding the gem rspec rails. However, when I run your written specs, I notice that there are some that are failing. Below is a screenshot of the run specs:

![image](https://github.com/IvonneBenitesRodriguez/add_recipe_app/assets/86318284/faef8d38-8f1d-40e0-8f05-4c34c99b6cdd)

**The specific files that are facing this failure include:**
1. spec/views/food_index_spec.rb
2. spec/views/public_recipes_spec.rb
3. spec/views/recipes_index_spec.rb
4. spec/views/recipes_show_spec.rb
5. spec/views/shopping_list_spec.rb

**KIndly ensure that you resolve all pending requirements before resubmitting for review.**

Reply:
Thank you @Nemwel-Boniface very nice to meet you,
We modified it as you requested, 
Bests,